### DISCUSSION TITLE
Clear event/history?

### DISCUSSION THREAD

**Discussion by TritonB7** [goauthentik/authentik#5109] [Q&A] [answered]
Is there an easy way to delete/cull Authentik Events/Logs? I'd like to also reset the Dashboard Overview statistics.

**Comment by RobinFrcd** [comment]
Same question here, I have more than 500k lines in the event logs, it seems it really slows down my whole Authentik instance.

**Comment by TommyTran732** [comment, marked as answer]
I did this with the postgres container and it worked:

```
TRUNCATE TABLE authentik_events_event CASCADE;
```

**Reply by TritonB7** [reply]
Thanks @TommyTran732 , that did the trick!
